roles/horizon/templates/etc/openstack-dashboard/local_settings.py

```diff
@@ -148,6 +155,10 @@
 
 OPENSTACK_HYPERVISOR_FEATURES = {
     'can_set_mount_point': False,
+
+    # NOTE: as of Grizzly this is not yet supported in Nova so enabling this
+    # setting will not do anything useful
+    'can_encrypt_volumes': False
```

Comment:
This may break our users's ability to use the volume encryption feature we've landed.